pkg/features/isis/collector.go

```diff
@@ -99,3 +111,24 @@ func (c *isisCollector) isisAdjancies(client collector.Client) (*adjacencies, er
 
 	return &adjacencies{Up: float64(up), Total: float64(total), Adjacencies: x.Information.Adjacencies}, nil
 }
+
+func (c *isisCollector) isisInterfaces(interfaces interfaces, ch chan<- prometheus.Metric, labelValues []string) {
+	for _, i := range interfaces.IsisInterfaceInformation.IsisInterface {
+		if strings.ToLower(i.InterfaceLevelData.Passive) == "passive" {
+			continue
+		}
+		labels := append(labelValues,
+			i.InterfaceName,
+			"",
```

Comment:
What is this empty string?

Reply:
Fixed, created new lables for this metric